comp/snmptraps/status/impl/status.go

```diff
@@ -24,16 +32,8 @@ func init() {
 	trapsExpvars.Set("PacketsAuthErrors", &trapsPacketsAuthErrors)
 }
 
-// Manager exposes the expvars we care about
-type Manager interface {
-	AddTrapsPackets(int64)
-	GetTrapsPackets() int64
-	AddTrapsPacketsAuthErrors(int64)
-	GetTrapsPacketsAuthErrors() int64
-}
-
-// New creates a new manager
-func New() Manager {
+// New creates a new component
+func New() status.Component {
```

Comment:
Can you have a more explicit name and not export the function?

Reply:
Done.